synapse/_scripts/review_recent_signups.py

```diff
@@ -121,6 +126,9 @@ def main() -> None:
         action="store_true",
         help="Only print user IDs that match.",
     )
+    parser.add_argument(
+        "-a", "--exclude-app-service", help="Exclude appservice users.", default=False
```

Comment:
For boolean flags you want to use this:
```suggestion
        "-a", "--exclude-app-service", help="Exclude appservice users.", action="store_true"
```

(Otherwise it is expecting you to write `-a true` instead of `-a`.)

```
$ synapse_review_recent_signups -c config.yaml -a
usage: synapse_review_recent_signups [-h] -c CONFIG_FILE [-s duration] [-e] [-u] [-a EXCLUDE_APP_SERVICE]
synapse_review_recent_signups: error: argument -a/--exclude-app-service: expected one argument
```